### ISSUE TITLE
Icons size

### ISSUE THREAD

**Issue by plegrand1** [sciancio/connectionmanager2#62]
Using that version on my debian sid with gnome-shell 3.26 works fine except the icons size. 
if i modify the stylesheet.css i can reduce the size but the big icons come back again and i dont know why ?
May be a solution here ?

**Comment by Borromini** [comment]
Are you seeing this behaviour after the screensaver kicks in? That's how it goes here at least... Only restarting Gnome Shell resets the icons to their normal size then.

**Comment by plegrand1** [comment]
Yes the problem appears just after screensaver. No way to solve this problem ?

**Comment by Borromini** [comment]
I can only say I have the same issue... Besides the workaround I don't know how to fix it.

**Comment by plegrand1** [comment]
I think i found a "solution" for this problem. But i cant explain what is the problem  ;-)

In the file : "~/.local/share/gnome-shell/extensions/connectionmanager2@ciancio.net/extension.js"
line 165 : 
							//style_class: 'connmgr-icon' });
							icon_size: 16 });
line 208 : 
							//style_class: 'connmgr-icon' });
							icon_size: 16 });

It seems to works fine now

**Comment by Borromini** [comment]
Great, thanks!
